pages/form.js

```diff
@@ -0,0 +1,162 @@
+import Navbar from '../components/Navbar';
+import { useState } from 'react';
+import { nanoid } from 'nanoid';
+import StyledButton from '../components/Button';
+import Title from '../components/Header';
+import { today } from '../utils/date';
+import { getAgeGroup } from '../utils/extract';
+import { useRouter } from 'next/router';
+
+export default function FormPage() {
+  const router = useRouter();
+  const [title, setTitle] = useState('');
+
+
+  function handleTitleChange(event) {
+    const newTitle = event.target.value.trimStart();
```

Comment:
you should consider whether you really want to remove only blanks at the beginning, or also those that may have been made unintentionally at the end. Accordingly, you would then only write .trim() :)